### PR TITLE
Improved working directory and import management

### PR DESCRIPTION
**Pull request by RichKMLS**
This pull request introduces the following improvements:

## Added
- ~Code to get the current working directory.~
- ~Code to get the expected working directory based on the location of invokeai_configure.py file.~
- ~Logic to switch to the expected working directory if it differs from the current one and to exit gracefully if an error occurs.~
## Changed
- The Imports to follow PEP 8 style guide more closely.
- The imports from the local invokeai package to use absolute paths.

### CODE REVIEW

**Comment by ebr** [comment]
Thank you for the contribution. Good call on replacing relative paths with package imports.

Could you please provide some context on what is being solved by the working directory change? What is the "expected" directory here?

The current expectation is that the config interface should work irrespective of the current working directory - is that breaking for you in some way?

Thank you!

File: invokeai/backend/config/invokeai_configure.py [hunk 1]
```diff
@@ -36,17 +44,34 @@
     CLIPTokenizer,
 )
 
+# Get the current working directory
+current_dir = os.getcwd()
+
+# get the expected working directory based on the file location
+expected_dir = Path(__file__).resolve().parent.parent.parent
```
[the remark this answers is not in the document]
Comment (originates) [reply invokeai/backend/config/invokeai_configure.py]: My mistake, changing the working directory didn't actually do anything productive so I removed it.

**Review by lstein — APPROVED** [review]
I'm OK with these changes.

**Comment by hipsterusername** [comment]
@RichKMLS - Can you share your thoughts on the above inquiries?

**Comment by lstein** [comment]
This PR seems to be inactive. There is an outstanding question with regard to the purpose of this change. @RichKMLS can you respond?

**Comment by Millu** [comment]
Closing this PR as inactive

**Pull request closed without merging**